pkg/querier/chunk_store_queryable_test.go

```diff
@@ -43,7 +45,10 @@ func TestChunkQueryable(t *testing.T) {
 		for _, testcase := range testcases {
 			for _, encoding := range encodings {
 				for _, query := range queries {
+					query := query
```

Comment:
```suggestion
```
Not necessary, query is already instantiated in the following line.

Reply:
@friedrichg removing this causes the "loop variable" warning. I understand that it's instantiated in line 49. Is the `query` in line `testRangeQuery(t, queryable, queryEngine, from, query)` loop variable or are we trying to access the **query field in query structure**?